libraries/chain/controller.cpp

```diff
@@ -3049,7 +3049,7 @@ bool controller::skip_db_sessions( ) const {
 }
 
 bool controller::skip_trx_checks() const {
-   return light_validation_allowed(my->conf.disable_replay_opts);
+   return light_validation_allowed(my->conf.force_all_checks);
```

Comment:
Can we just remove the arguments for `light_validation_allowed` now?

Reply:
Done.